server/src/main/java/io/littlehorse/server/streamsimpl/storeinternals/LHStoreWrapper.java

```diff
@@ -71,6 +80,7 @@ public void delete(String fullStoreKey) {
 
     public Bytes getRaw(String rawKey) {
         totalGets++;
+        log.warn("Getting: {}", rawKey);
```

Comment:
warn?

Reply:
Whoops.

Reply:
Anyways, I think we should not log (even trace) keys since the Tag keys for Variables might include sensitive data.